nifi-nar-bundles/nifi-apicurio-bundle/nifi-apicurio-schema-registry-service/src/main/java/org/apache/nifi/apicurio/schemaregistry/ApicurioSchemaRegistry.java

```diff
@@ -56,6 +57,15 @@ public class ApicurioSchemaRegistry extends AbstractControllerService implements
             .addValidator(StandardValidators.URL_VALIDATOR)
             .required(true)
             .build();
+    static final PropertyDescriptor SCHEMA_GROUP_ID = new PropertyDescriptor.Builder()
+            .name("Schema Group ID")
+            .displayName("Schema Group ID")
+            .description("The artifact Group ID for the schemas")
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .expressionLanguageSupported(ExpressionLanguageScope.ENVIRONMENT)
+            .defaultValue("default")
```

Comment:
For commiter:  "If you do not specify a unique group ID, you must specify ../groups/default in the API path."